app/src/main/java/com/github/se/cyrcle/ui/addParking/attributes/AttributesPicker.kt

```diff
@@ -180,11 +186,24 @@ fun AttributesPicker(
                         selectedValue = capacity,
                         label = stringResource(R.string.attributes_picker_capacity_label),
                     )
-                    EnumDropDown(
-                        options = ParkingRackType.entries.toList(),
-                        selectedValue = rackType,
-                        label = stringResource(R.string.attributes_picker_rack_type_label),
-                    )
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.fillMaxWidth()) {
+                          EnumDropDown(
+                              options = ParkingRackType.entries.toList(),
+                              selectedValue = rackType,
+                              label = stringResource(R.string.attributes_picker_rack_type_label),
+                              modifier = Modifier.weight(1f).fillMaxWidth())
+                          Icon(
```

Comment:
I think that such an Icon can and will be reused later in our app, would you be so kind as to turn it into an Atom ?

Reply:
I think we can do that when we plan to reuse it (for example as Julien mentionned, in the Parking Details Screen). Does that sound right to you?